app/models/requests/MovementRequest.scala

```diff
@@ -25,7 +25,6 @@ case class MovementRequest(
   consignmentReference: ConsignmentReferences,
   movementDetails: MovementDetailsRequest,
   location: Option[Location] = None,
-  arrivalReference: Option[ArrivalReference] = None,
```

Comment:
Should this be also propagated to the BE ?

Reply:
Backend is ready and merged